### ISSUE TITLE
Remove hard coded path in scripts

### ISSUE THREAD

**Issue by gfoo** [LaDHUL/oeuvres-roud#20]
https://github.com/LaDHUL/oeuvres-roud/blob/75ae73e4600b4d80df1d2da59df9d918c8fd54b4/load-onto.expect#L6

As explained in the comment of the scrpit, this should be: `spawn console.sh --force --echo --serverURL [lindex $argv 0]`

but it requires to add `console.sh` in the PATH, that is in your case before to launch the script:
```shell
export PATH=/home/espadini/Github/LaDHUL/Knora/webapi/scripts/openrdf-console/bin/:$PATH
# to test the PATH
echo $PATH
console.sh -v
```

**Comment by elespdn** [comment]
Done
https://github.com/LaDHUL/oeuvres-roud/commit/4ad98a6b4f4d32b46ba36a3c4a8b3e53b872a96f

je peux fermer ?

**Comment by gfoo** [comment]
oui. 

Pour fermer automatiquement une issue lors d'un commit tu peux mettre `closes #num`

Par exemple pour celui-ci :   `closes #20: remove path from load onto`

**Comment by elespdn** [comment]
closes #20: remove path from load onto
